vscode/test/suites/extensionUtils.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as vscode from "vscode";
+import { type ExtensionApi } from "../../src/extension";
+
+// Q# extension log level. Increase this for debugging.
+const extensionLogLevel = "warn";
+
+export async function activateExtension() {
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const ext = vscode.extensions.getExtension("quantum.qsharp-lang-vscode-dev")!;
+  if (ext.isActive) {
+    return;
+  }
+
+  const start = performance.now();
+  const extensionApi: ExtensionApi = await ext.activate();
+  const logForwarder = extensionApi.logging;
+  if (!logForwarder) {
+    throw new Error(`qsharp-tests: extension did not return a log forwarder`);
+  } else {
```

Comment:
Maybe a naive comment since I don't know ts that well. Is the `else` needed? Since an error is thrown inside the `if` block already, could the logic inside the `else` block just be part of the function block?

Reply:
IMO it is more legible having this logic outside the `else` block, but it is just a preference and there might be some reason for doing it this way that I don't know about so feel free to ignore this suggestion if it doesn't make sense.

Reply:
I totally agree, thanks for the suggestion.